### DISCUSSION TITLE
fallback.skip

### DISCUSSION THREAD

**Discussion by marchino2000** [savonet/liquidsoap#2455] [User Support]
Hi everybody

```
hitlist    = playlist(reload_mode="watch","hitlist.m3u")

jingles    = playlist(reload_mode="watch","jingles.m3u")

dc = fallback.skip(fallback=delay(120.,jingles), hitlist)
```

I'm doing some tests with liquidsoap 2.0.5
I expected from the above code to have jingles played every 120 seconds (not track sensitive), but only hitlist is actually played. Is there anything wrong with my script?

Thank you!

**Comment by toots** [comment]
I think it should be the other way:
```ruby
hitlist    = playlist(reload_mode="watch","hitlist.m3u")

jingles    = playlist(reload_mode="watch","jingles.m3u")

dc = fallback.skip(fallback=hitlist, delay(120.,jingles))
```

**Reply by marchino2000** [reply]
It was that simple and I didn't think about it... ;) thank you Toots!
